ddtrace/internal/rate_limiter.py

```diff
@@ -13,8 +13,8 @@ class RateLimiter(object):
 
     __slots__ = (
         "_lock",
-        "current_window",
-        "last_update",
+        "current_window_ns",
```

Comment:
The change to ns is good but adds overhead to this PR. Can we do it separately?

Reply:
I moved some of the non-performance related changes to another PR to reduce the size of this one, happy to then break this into 2 more PRs:

1. Perf improvements to DatadogSampler.sample
2. Perf improvements to RateLimiter

What do you think?

Reply:
👍 smaller the better IMO